Sources/Support/DebugUI/DebugViewModel.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  DebugViewModel.swift
+//
+//  Created by Nacho Soto on 5/30/23.
+
+import Foundation
+
+#if DEBUG && os(iOS) && swift(>=5.8)
+
+import SwiftUI
+
+@MainActor
+@available(iOS 16.0, *)
+final class DebugViewModel: ObservableObject {
```

Comment:
Should we exclude these new files from coverage?

Reply:
I want to write UI snapshot tests for this since it's a public facing feature :)

Reply:
Done: #2630